local-test-configuration/scripts/initialize-chain.ts

```diff
@@ -54,8 +55,7 @@ export const deriveRole = (adminRole: string, roleDescription: string) => {
 };
 
 function encodeUpdateParameters() {
-  const HUNDRED_PERCENT = 1e8;
-  const deviationThresholdInPercentage = HUNDRED_PERCENT / 100; // 1%
+  const deviationThresholdInPercentage = HUNDRED_PERCENT / 100n; // 1%
```

Comment:
Importing the already defined constant satisfies the lint that caused initial CI failure: `You must place screaming snake case at module scope. If this is not meant to be a module-scoped variable, use camelcase instead  @shopify/prefer-module-scope-constants`.